resources/build/build-utils.sh

```diff
@@ -137,3 +137,39 @@ findTier
 findVersion
 # printVersionUtilsDebug
 printBuildNumberForTeamCity
+
+# Intended for use with macOS-based builds, as Xcode build phase "run script"s do not have access to important
+# environment variables.  Doesn't hurt to run it at other times as well.  The output file is .gitignore'd.
+function exportEnvironmentDefinitionScript() {
+    ENVIRONMENT_SH="$KEYMAN_ROOT/resources/environment.sh"
+
+    # Remove old copy if it exists
+    rm $ENVIRONMENT_SH > /dev/null 2> /dev/null 
```

Comment:
This fails if the file does not exist (also the variable should be quoted!)

```suggestion
    [ -f "$ENVIRONMENT_SH" ] && rm "$ENVIRONMENT_SH" 
```

(This is currently breaking the trigger for test builds for 2782 and 2771).